src/main/java/org/jitsi/jicofo/recording/jibri/JibriRecorder.java

```diff
@@ -197,10 +197,19 @@ public void init()
     @Override
     public void dispose()
     {
-        XMPPError error = sendStopIQ();
-        if (error != null)
+        try
+        {
+            XMPPError error = sendStopIQ();
+            if (error != null)
+            {
+                logger.error(
+                    "An error response to the stop request: " + error.toXML());
+            }
+        }
+        catch (OperationFailedException e)
         {
-            logger.error("Error when sending stop request: " + error.toXML());
+            logger.error("Failed to send stop IQ - XMPP disconnected", e);
+            recordingStopped(null, false /* do not send any status updates */);
```

Comment:
Is it intentional that we call recordingStopped on exeption, but not when "error != null" above?